<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>method3主程序</string>
  </property>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>113</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>交叉率：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pc">
      <property name="text">
       <string>1.0</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>1.0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>291</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>智能体数目：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="agent_num">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>23</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NSGA-III参数：</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>173</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>变异方法：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="eta_c">
      <property name="text">
       <string>多项式变异</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>多项式变异</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>63</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>超平面等分数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="p1_">
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>261</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>规模设置：(需要注意的是测试数据和程序是分离的，请注意导入)</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>291</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>任务数目：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="task_num">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>211</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>迁移个数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="migratenum">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>361</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>231</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>迁移学习参数：</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>111</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>迭代次数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="iter_num">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>173</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>变异分布索引：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="eta_m">
      <property name="text">
       <string>20</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>20</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>53</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>算法名称：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name">
      <property name="text">
       <string>NSGAIII</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>NSGA-III</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
